### DISCUSSION TITLE
Cloudwatch: Cross Region Cross Account support

### DISCUSSION THREAD

**Discussion by Alan01252** [grafana/grafana#69440] [Data Sources]
When using cloudwatch monitoring account I want to be able to select the region to be able to select the metrics from as I can within the cloudwatch console.

![image](https://github.com/grafana/grafana/assets/112859/da3d2702-63c0-4261-a7d6-3069efe5e1b2)

In grafana currently I only get a list of accounts when choosing the region the monitoring account is in

![image](https://github.com/grafana/grafana/assets/112859/153a37ad-7a1d-4342-a7e1-f6cc451d02fb)

If I choose a different region the account list vanishes and I can't select metrics from that account.

**Comment by Alan01252** [comment]
The workaround is to configure a new Cloudwatch monitoring setup in the region of your choice and link back to that account. That does work, but it's a tiny bit more cognitive overhead.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82638. Feel free to continue the discussion around this there. Thank you!
